site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -283,9 +280,8 @@ const TerminalPage: FC = () => {
     command,
     proxy.preferredPathAppURL,
     reconnectionToken,
-    terminal,
```

Comment:
Without `terminal` here the effect will not run again after we set it, right?  So we could end up never connecting the terminal, which might explain the failing test.

Reply:
Ahhh ok, now I see why the terminal was a state and not a ref. Thanks